src/pages/Register/Register.jsx

```diff
@@ -20,25 +21,38 @@ function Register({ navigation }) {
       <View className="flex-[3] justify-center items-center">
         <Image source={logo} className="w-[200] h-[75]" contentFit="fill" />
       </View>
-
+      <Formik
+     initialValues={{ nameAndSurname: '',email: '',password: '' ,passwordConfirm: ''}}
```

Comment:
Instead of writing all properties of initialValues, you can initialize all properties at one object and declare it initialValues property at Formik